.github/workflows/build-and-run-model.yaml

```diff
@@ -6,11 +6,12 @@
 # that the build continues to work properly, but Batch jobs are gated behind
 # a `deploy` environment that requires manual approval from a
 # @ccao-data/core-team member.
-#
+
 name: build-and-run-model
 
 on:
   pull_request:
+    types: [opened, reopened, synchronize, closed]
```

Comment:
Per [the docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request), `opened`, `reopened`, and `closed` are the default enabled event types when `pull_request` is enabled without a `types` attribute (as was the case before this PR). Now that the `build-and-run-batch-job` workflow supports running a cleanup step on the `closed` event ([see docs here](https://github.com/ccao-data/actions/blob/jeancochrane/add-batch-and-terraform-workflows-and-actions/README.md#build-and-run-batch-job)), we need to specify all of the previously-implicit event types in order to add `closed` to the list.